sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -728,67 +575,412 @@ const alwaysBeExistingRule = "alwaysbeexistingrule";
 
       // None of below work
       // To be investigated further as part of https://github.com/azure/azure-sdk-for-js/issues/5354
-      defaultRuleDescription: {
-        Name: "test1",
-        Filter: { SqlExpression: "1=1", CompatibilityLevel: "20" }
-      },
-      messageCount: 5,
-      enablePartitioning: true,
-      maxSizeInMegabytes: 2048,
-      sizeInBytes: 500
+      defaultRuleDescription: {
+        Name: "test1",
+        Filter: { SqlExpression: "1=1", CompatibilityLevel: "20" }
+      },
+      messageCount: 5,
+      enablePartitioning: true,
+      maxSizeInMegabytes: 2048,
+      sizeInBytes: 500
+    },
+    output: {
+      lockDuration: "PT5M",
+      maxDeliveryCount: 20,
+      defaultMessageTtl: "P2D",
+      autoDeleteOnIdle: "PT1H",
+      deadLetteringOnFilterEvaluationExceptions: false,
+      deadLetteringOnMessageExpiration: true,
+      enableBatchedOperations: false,
+      requiresSession: true,
+
+      forwardDeadLetteredMessagesTo: undefined,
+      defaultRuleDescription: undefined,
+
+      messageCount: 0,
+      enablePartitioning: undefined,
+      maxSizeInMegabytes: undefined,
+      sizeInBytes: undefined,
+
+      forwardTo: undefined,
+      userMetadata: undefined,
+      messageCountDetails: undefined,
+      entityAvailabilityStatus: "Available",
+      status: "Active",
+
+      subscriptionName: "alwaysBeExistingSubscription2",
+      topicName: "alwaysBeExistingTopic1"
+    }
+  },
+  {
+    subscriptionName: "alwaysBeExistingSubscription20",
+    topicName: "alwaysBeExistingTopic1",
+    testCaseTitle: "pass in forwardDeadLetteredMessagesTo",
+    input: {
+      lockDuration: "PT5M",
+      maxDeliveryCount: 20,
+      forwardDeadLetteredMessagesTo: "alwaysBeExistingTopic1",
+      forwardTo: "alwaysBeExistingTopic2",
```

Comment:
Looks like we have multiple comments on here about testing absolute URLs - this isn't possible since the endpoint being used in the URL would be CI specific and unknown.
Hence if you see the updated tests, its now checking for the output URL to contain the expected entity name instead.

Can add a comment inline to elaborate..

Reply:
Updated tests to check for absolute URLs and added in new tests as well. @ramya-rao-a